js/app/src/components/StatusTracker/ToastContent.svelte

```diff
@@ -0,0 +1,173 @@
+<script lang="ts">
+	import { Alert, Information, Warning } from "@gradio/icons";
+	import { createEventDispatcher, onMount, afterUpdate } from "svelte";
+	import { fade } from "svelte/transition";
+	import type { ThemeMode } from "../types";
+
+	export let message: string = "";
+	export let description: string = "";
+	export let type: string = "";
```

Comment:
```suggestion
export let type: ToastMessage["type"];
```
can't be any arbitrary string